ui/e2e/playwright.config.ts

```diff
@@ -0,0 +1,122 @@
+// Copyright 2022 The Perses Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import path from 'path';
+import type { PlaywrightTestConfig } from '@playwright/test';
+import { devices } from '@playwright/test';
+
+const IS_CI = process.env.CI;
```

Comment:
Just a "future" thought/possible improvement--wonder if it would be clearer to split into multiple config files like we do for the webpack configs rather than keeping the branching logic in this one file?

Reply:
I'll take a pass at extracting it in a follow-up PR, so we can get this initial setup through to unblock writing more tests.